graph.py

```diff
@@ -0,0 +1,81 @@
+from linkedlist import LinkedList
+
+
+class Graph:
+    def __init__(self):
+        self._storage = {}
+
+    def nodes(self):
+        """Returns a list of all the nodes in the graph/"""
+        return [node for node in self._storage]
+
+    def edges(self):
+        """Returns a list of all the edges in the graph."""
+        return [edges for edges in self._storage.values()]
```

Comment:
These are lists of nodes.

If your graph looks like:
```
{"a": {"b", "c"}}
```
Then your edges would look like:
```
[("a", "b"), ("a", "c")]
```

Reply:
but my graph is like this
{"a": {("a", "b"), ("a", "c")}}

Reply:
I see.  That's probably fine.